### DISCUSSION TITLE
Feature Request

### DISCUSSION THREAD

**Discussion by sambickeita** [AI4Finance-Foundation/FinGPT#5] [Ideas]
Would it be possible to add to FInGPT the ability to connect to API keys ?

**Comment by LucasMartinCalderon** [comment]
Great work here, and thank you for putting this together. 

I've been working with LangChain and LLMs for a while now, which can potentially make FinGPT more flexible, knowledgeable and smarter, happy to offer some open source contributions. Are you planning to play with such libraries for the system?

Thanks.

**Reply by YangletLiu** [reply]
Thanks!  Yes, we are quite interested in LangChain, and the new Falcon LLM.
Our team are clearning financial data, and then connect with LLMs, providing demos to different open-source community.
We very much appreciate your input, if you would like to give us a hand.

**Comment by kattapug** [comment]
To add to this Autogen is another such libraries worth exploring.
